src/components/layout/nav/MainNavMenu.tsx

```diff
@@ -1,25 +1,29 @@
 import React from 'react'
-import { Button, Grid } from '@mui/material'
+import { Grid, Typography } from '@mui/material'
 import { useTranslation } from 'next-i18next'
 
 import { routes } from 'common/routes'
 
 import DonationMenu from './DonationMenu'
 import ProjectMenu from './ProjectMenu'
+import LinkButton from 'components/common/LinkButton'
 
 export default function MainNavMenu({ children }: { children?: React.ReactNode }) {
   const { t } = useTranslation()
 
   return (
     <Grid container direction="row" wrap="nowrap" alignItems="baseline" spacing={4}>
       <Grid item>
-        <Button
+        <LinkButton
           variant="outlined"
+          size="large"
           color="inherit"
-          href={routes.campaigns.index}
-          style={{ borderColor: '#32A9FE' }}>
-          {t('nav.donatе')}
-        </Button>
+          style={{ borderColor: '#32A9FE' }}
```

Comment:
@Katzarov Let's use `sx` instead of `style` here - it compiles the CSS as class instead of passing it inline

```suggestion
          sx={{ borderColor: '#32A9FE' }}
```